yapapi/rest/payment.py

```diff
@@ -130,12 +130,14 @@ def new_allocation(
         """Creates new allocation.
 
         - `amount`:  Allocation amount.
-        - `expires`: expiration timestamp. by default 30 minutes from now.
+        - `expires`: expiration timestamp. set to default 10 years as a work-around.
         - `make_deposit`: (unimplemented).
 
         """
         now = datetime.now(timezone.utc)
-        allocation_timeout: datetime = expires or now + timedelta(minutes=30)
+        # as a work-around, we're setting the allocation timeout into far future
+        # @todo http://link-to-the-issue
```

Comment:
please add an issue and replace this with an actual link to the issue calling for a refactoring of the relevant functionality ... or just link to the issue that originally reported this problem...

Reply:
Fixed.